metadata-db-app/src/cmr/metadata_db/data/oracle/collection_table.clj

```diff
@@ -104,6 +105,9 @@
   ;; Need this for transaction-id post commit check
   (j/db-do-commands db (format "CREATE INDEX %s_crtid ON %s (concept_id, revision_id, transaction_id)"
                                table-name
+                               table-name))
+  (j/db-do-commands db (format "CREATE INDEX %s_c_i ON %s (created_at)"
```

Comment:
It is not used right now, but I can see us using it the future for batching or something similar. If you want us to remove it we can remove it. Collection tables are small enough that I don't think it is adding significant overhead to keep it.